aggregator/flush_mgr_options.go

```diff
@@ -35,7 +37,11 @@ const (
 )
 
 var (
-	defaultWorkerPoolSize = int(math.Max(float64(runtime.NumCPU()/8), 1.0))
+	defaultWorkerPoolSize         = int(math.Max(float64(runtime.NumCPU()/8), 1.0))
+	defaultFlushTimesKeyFormat    = "/shardset/%s/flush"
+	defaultFlushTimesPersistEvery = 10 * time.Second
+	defaultMaxNoFlushDuration     = 15 * time.Minute
```

Comment:
Maybe this should be 5 minutes by default?  Or even 2 minutes.  That's pretty far behind for all metrics to be behind by if leader goes down no?  Or would it auto-elect itself before that time?